### ISSUE TITLE
Allow 'aria-owns' to pass when used on collapsed combobox (i.e aria-expanded="false")

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1524]
Related to #1463

Expectation: No ```aria-valid-attr-value``` failure when the child element referenced to ```aria-owns``` on a collapsed ```combobox```(i.e with aria-expanded="false") is not present in the DOM

Actual: ```'aria-valid-attr-value'``` fails ```aria-owns``` on a collapsed combobox when it is referenced to a child element not in the DOM 

Motivation: This should be allowed as there would be no impact to the behavior of the combobox from AT point of view

<pre><code>
axe-core version: 3.2.2
axe-Coconut version: 3.8.0
</code></pre>

**Comment by WilcoFiers** [comment]
Good point. Pretty similar to a thing we fixed recently.

**Comment by chandana7393** [comment]
Verifed behaving as expected.
![image](https://user-images.githubusercontent.com/45285854/58473178-97cd3980-8165-11e9-87cc-012ebafc6caa.png)
**Tested Environment:**
Axe-coconut - 3.8.0.14704v
Chrome - 74v
OS - Windows 10 64 bit.
